db/fixtures/miq_user_roles.yml

```diff
@@ -352,6 +358,8 @@
   - ems_physical_infra_show_list
   - ems_physical_infra_tag
   - ems_physical_infra_timeline
+  - event_stream_show_list
```

Comment:
Gabriel can you explain more what you mean?  Are you looking for additional spec tests or something else?